assets/src/scss/pages/_page.scss

```diff
@@ -89,12 +89,71 @@ div.page-template {
   }
 }
 
-.error-message {
-  color: $orange-hover;
-}
+.password-form {
+  .input-text{
+    font-size:16px;
+    line-height:1.6;
+    font-weight:400;
+    margin-bottom:16px;
+    position:relative;
+  }
+
+  .input-text input{
+    border:1.5px solid $grey-20;
```

Comment:
The border width should also be 1px here, at the moment it looks a bit weird on hover

Reply:
Yes, I agree with Maud